src/pages/settings/settings.html

```diff
@@ -599,15 +599,24 @@
                                     <tr>
                                         <td>
                                             <span
-                                                i18n="i18n_LABEL_SETTINGS_TAB_PLAYBACK_AutoClick_SkipAd"
+                                                i18n="i18n_LABEL_SETTINGS_TAB_PLAYBACK_AUTOCLICK_SKIPAD"
                                             ></span>
+                                                i18n="i18n_LABEL_SETTINGS_TAB_PLAYBACK_SURROUND_SOUND"
```

Comment:
🤔 I think some issues have occurred here with the merging of a few PRs last night for the Surrounds Sound.
Looks like they've both overlapped each other as values in the settings.

Reply:
Noticed I'll fix it when I have time